controllers/comments.controller.js

```diff
@@ -40,11 +39,36 @@ const postCommentToArticle = (req, res, next) => {
       res.status(201).send(comment);
     })
     .catch((error) => {
-      if (error.status === 400 && error.msg === "(username) and (body) are required fields.") {
+      if (
+        error.status === 400 &&
+        error.msg === "(username) and (body) are required fields."
+      ) {
         return res.status(400).send(error);
       }
       next(error);
     });
 };
 
-module.exports = { getCommentsByArticleIdLifo, postCommentToArticle };
+const deleteComment = (req, res, next) => {
+  const commentId = req.params.comment_id;
+  
+  checkIfCommentExists(commentId)
+    .then((exists) => {
+      if (!exists) {
+        return res.status(404).send({ msg: "Non-existent Comment ID" });
```

Comment:
I think you should be Promise.rejecting and allowing your error handling middleware to handle this :)